packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -824,15 +824,15 @@ abstract class ResidentRunner {
 }
 
 class OperationResult {
-  OperationResult(this.code, this.message, [this.hint]);
+  OperationResult(this.code, this.message, { this.hint });
 
   final int code;
   final String message;
   final String hint;
 
   bool get isOk => code == 0;
 
-  static final OperationResult ok = new OperationResult(0, '');
+  static final OperationResult kOk = new OperationResult(0, '');
```

Comment:
you can omit the `k` as of the recent style guide updates, since it's not a global

Reply:
done